### ISSUE TITLE
modified model with sanic serving 

### ISSUE THREAD

**Issue by samruddha-kf** [massquantity/LibRecommender#444]
Hi, 
I needed an extra mapping of items - item_category, which I added as an attribute of the model, which I will be using later to filter the recommendations. Right now I have made no code changes to filter, only added the attribute to the model and saved it. This model after loading with libserving module, I could use with tf_serving method. i.e. I can get the logits outputs with RESTful API tf_serving predict method. But when I try to use the sanic framework to get the n recommendations I am getting an error. The error is at line 182 of tf_deploy.py. The module is not able to load the model. What can I do to rectify this?

**Comment by massquantity** [comment]
Maybe there is a name conflict. Please provide your code. 

You can also print [data](https://github.com/massquantity/LibRecommender/blob/master/libserving/sanic_serving/tf_deploy.py#L177) here to check if it has the correct format.
